coral/src/app/components/PreviewBanner.tsx

```diff
@@ -34,13 +34,10 @@ function PreviewBanner({ linkTarget }: { linkTarget: string }) {
       marginBottom={"l1"}
       aria-label={"Preview disclaimer"}
     >
-      <Icon
-        aria-hidden={true}
-        color={"info-50"}
-        fontSize={20}
-        icon={infoSign}
-      />
-      <Typography variant={"body-small"}>
+      <Typography.SmallText>
+        <Icon icon={infoSign} color={"info-50"} style={{ marginTop: "5px" }} />
```

Comment:
`Icon` has `aria-hidden` attribute by default now, I presume ^^

Reply:
It already had :)